Source/ThirdParty/ANGLE/ANGLE.xcodeproj/project.pbxproj

```diff
@@ -3879,7 +4592,545 @@
 			isa = PBXSourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				4458D1272B635E1000B64F47 /* translator_fuzzer.cpp in Sources */,
+				7BF237752C80432B007D4EBC /* WebKit/TranslatorFuzzer.cpp in Sources */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		7BFBD7A72C7DE55300BFCA94 /* Sources */ = {
+			isa = PBXSourcesBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				7BFBD8DC2C7E025300BFCA94 /* AddExplicitTypeCasts.cpp in Sources */,
+				7BFBD7EC2C7DE96D00BFCA94 /* aligned_memory.cpp in Sources */,
+				7BFBD7E92C7DE95100BFCA94 /* angle_version_info.cpp in Sources */,
+				7BFBD8682C7E01D100BFCA94 /* angleutils.cpp in Sources */,
+				7BFBD8A92C7E023200BFCA94 /* AstHelpers.cpp in Sources */,
```

Comment:
Doesn't this basically duplicate (three times!) all of the source files used to compile libANGLE-shared.dylib?

It's a shame we have to duplicate targets instead of re-using them.

That's why I created `libJavaScriptCore.a` in the `JavaScriptCore` project—so that I didn't have to create a separate target to build all the source files twice.

The same thing could be done here, although it's a bit gnarly to get it right, so this is probably the best option for now, but it's going to break frequently (unless you commit to testing the build every time ANGLE is updated from upstream).

TL;DR: No need to make a change for this PR, but long term I'd like to see a `libANGLE.a` output that is then linked into `libANGLE-shared.dylib` if you need both static and dynamic linking of the code.